docs/ja/migration/compatibility-matrix.md

```
@@ -73,58 +76,58 @@ outline: deep
 | `deleteAll` クエリパラメータ | ✅ | ✅ | すべてのインスタンスを削除 |
 
-### NGSI-LD 出力フォーマット
+### NGSI-LD 出力形式
 
 | 機能 | GeonicDB | FIWARE Orion-LD | 備考 |
 |---------|:------------------:|:---------------:|-------|
-| normalized | ✅ | ✅ | 完全フォーマット (デフォルト) |
-| concise | ✅ | ✅ | 簡潔フォーマット (type 省略) |
+| normalized | ✅ | ✅ | 完全形式(デフォルト) |
+| concise | ✅ | ✅ | 簡潔形式(type 省略) |
 | keyValues / simplified | ✅ | ✅ | 値のみ |## クエリ機能
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)